packages/app/features/dashboard/profile/profileRow.tsx

```diff
@@ -6,13 +6,13 @@ import { tw } from "app/design-system/tailwind";
 type ProfileRowProps = {
   icon: ReactElement;
   title: string;
-  trailingText?: string;
+  trailingNumber?: number;
```

Comment:
Is there a reason why you've changed that?

Reply:
We want to display nothing when the number of items equals 0. It was a lot cleaner using number instead of string.